### PR TITLE
Expose terraform-aws-s3-log-storage module variables

### PR DESCRIPTION
**Pull request by tegamckinney**
This PR exposes the module variables from terraform-aws-s3-log-storage module.

Related #8

### CODE REVIEW

**Comment by tegamckinney** [comment]
@aknysh Is this what you were looking for to resolve #8?

File: variables.tf [hunk 1]
```diff
@@ -47,3 +47,58 @@ variable "force_destroy" {
   description = "A boolean that indicates the bucket can be destroyed even if it contains objects. These objects are not recoverable"
   default     = "false"
 }
+
+variable "versioning_enabled" {
+  description = "A state of versioning. Versioning is a means of keeping multiple variants of an object in the same bucket."
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "A state of versioning. Versioning is a means of keeping multiple variants of an object in the same bucket"
```

File: variables.tf [hunk 2]
```diff
@@ -47,3 +47,58 @@ variable "force_destroy" {
...
+  default     = "true"
+}
+
+variable "lifecycle_rule_enabled" {
+  description = "Enable lifecycle events on this bucket"
+  default     = "true"
+}
+
+variable "lifecycle_prefix" {
+  description = "Prefix filter. Used to manage object lifecycle events."
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "Prefix filter. Used to manage object lifecycle events"
```

File: variables.tf [hunk 3]
```diff
@@ -47,3 +47,58 @@ variable "force_destroy" {
...
+  default     = ""
+}
+
+variable "lifecycle_tags" {
+  description = "Tags filter. Used to manage object lifecycle events."
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "Tags filter. Used to manage object lifecycle events"
```

File: variables.tf [hunk 4]
```diff
@@ -47,3 +47,58 @@ variable "force_destroy" {
...
+  default     = {}
+}
+
+variable "noncurrent_version_expiration_days" {
+  description = "Specifies when noncurrent object versions expire."
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "Specifies when noncurrent object versions expire"
```

File: variables.tf [hunk 5]
```diff
@@ -47,3 +47,58 @@ variable "force_destroy" {
...
+  default     = "90"
+}
+
+variable "noncurrent_version_transition_days" {
+  description = "Specifies when noncurrent object versions transitions"
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "Specifies when noncurrent object versions transition"
```

**Comment by tegamckinney** [comment]
> thanks @tegamckinney
> Looks good.
> Just some minor nitpicks.
> Please rebuild README again after that:
> 
> ```
> make init
> make readme/deps
> make readme
> ```

@aknysh Updates completed.  Also - really like the build-harness toolchain; really nice!

**Pull request merged**